css/style.css

```
@@ -0,0 +1,11 @@
+#navbar
+{
+   position: fixed;
+   top: 0;
+}
+
+@media only screen and (max-width: 450px){
+body{

```

🙌 I kindly recommend you leave a space between the property and the value, so we respect good practices and improve the readability of our code 📑
```suggestion
body {

```